packages/app/src/elements/settings-security.ts

```diff
@@ -730,6 +742,42 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
         `;
     }
 
+    private _renderEmailNotifications() {
+        return html`
+            <div class="box">
+                <h2 class="padded uppercase bg-dark border-bottom semibold">${$l("Email Notifications")}</h2>
+
+                <div>
+                    <pl-toggle-button
+                        class="transparent"
+                        id="failedLoginAttemptsNotificationsToggle"
+                        .active=${app.account?.settings.notifications.failedLoginAttempts || false}
+                        .label=${html`<div class="horizontal center-aligning spacing layout">
+                            <pl-icon icon="weak"></pl-icon>
```

Comment:
Can we find a better icon for this please? 🙃 The envelope one should be fine if you don't want to get creative.

Reply:
Oops, sorry! I forgot about that.